protos/flyteidl/plugins/mpi.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package flyteidl.plugins;
+
+option go_package = "github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/plugins";
+
+// Custom proto for plugin that enables distributed training using https://github.com/kubeflow/mpi-operator
+message DistributedMPITrainingTask {
+  // number of worker spawned in the cluster for this job
+  int32 workers = 1;
+
+  // number of launcher replicas spawned in the cluster for this job
+  int32 launcher_replicas = 2;
```

Comment:
Is there a relationship between number of workers and launcher_replicas?

Naming-wise, should they be `workers` and `launchers` or `worker_replicas` and `launcher_replicas` ?

Reply:
no in operator both are replica. They are part of a common spec but both can have different value  https://github.com/kubeflow/mpi-operator/blob/a566d1d18046fe7f756b0ec7004693c898d589a0/pkg/apis/kubeflow/v1/types.go#L53